docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -199,10 +190,10 @@ To enable shipping traces to the SAP Cloud Logging service instance, follow this
 
    > [!WARNING]
    > Be cautious when you configure the **randomSamplingPercentage**:
-   > - Traces might consume a significant storage volume in Cloud Logging Service.
+   > - Traces might consume a significant storage volume in SAP Cloud Logging.
    > - The Kyma trace collector component does not scale automatically.
```

Comment:
```suggestion
   > - The Kyma trace collector component does not scale automatically. For manual scaling, see [Troubleshooting](https://help.sap.com/docs/btp/sap-business-technology-platform-internal/traces?state=DRAFT&version=Internal#loiof98cda5d058e48ff808ade541a64a6ad__section_kyma_traces_troubleshooting).
```
The link needs to be changed though...

Reply:
Meanwhile, Icke decided we drop the whole Caution note :D